### ISSUE TITLE
EViT-DeiT-S Finetune-30 Reproduce 

### ISSUE THREAD

**Issue by BWLONG** [youweiliang/evit#9]
Thanks for your open source work.

![image](https://user-images.githubusercontent.com/38601195/174768831-3437727f-6089-4e7e-858a-3096232fb7a9.png)

Following the finetune.sh configuration, I just use the batch_size of 8x256=2048, but the accuracy is only 79.35%.

**Comment by youweiliang** [comment]
Hi, which checkpoint did you use for finetuning? Did you modify any hyperparameters? I checked it twice for reproducibility and there should be no problem in getting 79.5% accuracy.

**Comment by BWLONG** [comment]
I used the batch_size of 8x256=2048 and the checkpoint=deit_small_patch16_224-cd65a155.pth , the other hyperparameters followed the finetune.sh configuration.

**Comment by ljmtendero** [comment]
Did you finally manage to reproduce their results? @BWLONG
